docs/enterprise-guide/dashboard-subscriptions.md

```diff
@@ -0,0 +1,28 @@
+
+## Customize filter values for each dashboard subscription
+
+This page covers the Enterprise Edition features for dashboard subscriptions. Our User's Guide can give you an introduction to [dashboard subscriptions](../users-guide/dashboard-subscriptions.md). 
+
+Enterprise Edition lets you set up multiple subscriptions for a single dashboard, and lets you customize which filter values to apply to each subscription. That way you can send one group of people an email (or Slack message) the contents of the dashboard with one filter value applied, and set up another subscription to send the dashboard's contents with a different filter applied. You only need to maintain one dashboard, and you can set up different subscriptions to give different groups of people the info they need. 
+
+### Adding a subscription
+
+To add a subscription, click on the **+** icon in the dashboard subscription panel.
+
+![Dashboard with multiple subscriptions](./images/dashboard-subscriptions/multiple-subscriptions.png)
+
+Here we have one dashboard with two subscriptions set up: the first subscription sent to Example Person with "VT" as the value for the State Filter, and the second sent to Other Person with "CA" as the State filter value. Both subscriptions lack a value to apply to the Created_At filter.
+
+### Setting filter values
+
+You can set values for each filter on the dashboard. Any values you set for the subscription will override the dashboard's default filter values. If you don't set a filter value for the subscription, the dashboard will run with the default filter values applied, if any are set.
```

Comment:
I think you can simplify this here, 'cause for EE the default values will be even more obvious since they'll be displayed in the list of configurable filters for the subscription.

```suggestion
You can set values for each filter on the dashboard. If you have any dashboard filters with default values, you can override those defaults for a given subscription, or leave them as-is.
```